Thanks @Jake-Carter .   I tried building the example projects I included and looks like a couple things are off with the updates.  Rather than creating the tinyusb.a library, the library name is the project name.  And similarly, the project's ELF file is trying to be placed in the TUSB build folder.  I can hunt this down next week if you don't have the time, but figured you just may know it off the top of your head.  Let me know.

```
Loaded project.mk
****************************************************************************
* Analog Devices MSDK
* v2023_10-172-gd3f8669fd6
* - User Guide: https://analogdevicesinc.github.io/msdk/USERGUIDE/
* - Get Support: https://www.analog.com/support/technical-support.html
* - Report Issues: https://github.com/analogdevicesinc/msdk/issues
* - Contributing: https://analogdevicesinc.github.io/msdk/CONTRIBUTING/
****************************************************************************
make -C C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb lib BUILD_DIR=C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb/build/MAX32650
Loaded /c/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb/project.mk
- AR C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb/build/MAX32650/cdc_msc.a
- LD C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb/build/MAX32650/cdc_msc.elf
arm-none-eabi-gcc.exe: error: C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb/build/MAX32650/tinyusb.a: No such file or directory
make: *** [C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/CMSIS/Device/Maxim/GCC/gcc.mk:594: C:/Workspace/MSDK_Development/brentk-adi-fork/Libraries/tinyusb/build/MAX32650/cdc_msc.elf] Error 1

 *  The terminal process "C:\windows\System32\cmd.exe /d /c make -r -j 8 --output-sync=target --no-print-directory TARGET=MAX32650 BOARD=EvKit_V1 MAXIM_PATH=C:/Workspace/MSDK_Development/brentk-adi-fork MAKE=make PROJECT=cdc_msc" terminated with exit code: 2. 
 ```

--- a/Libraries/tinyusb/project.mk
+++ b/Libraries/tinyusb/project.mk
@@ -1,6 +1,6 @@
 include tinyusb.mk
 
-PROJECT := tinyusb
+override PROJECT := tinyusb
 
 LIB_PERIPHDRIVERS := 1
 

